CODE_REVIEW.md

```diff
@@ -0,0 +1,33 @@
+# Code Review  
+## Code Smells  
+- action naming should be consistent - three types of actions (commands, documents, events) should each follow naming convention - a command should start with a verb (like 'ADD_ITEM'), a document should be a noun (like 'ITEM'), an event should use a past tense verb (like 'APP_WENT_ONLINE')  
+- could create a new component for books grid and each grid item to be used in the book-search component  
+- book-search component template could use angular built-in date pipe instead of formatDate function  
+- could use async pipe in the book-search component template instead of using subscribe in component, like in reading-list component and total-count component  
+- no loading indicator  
+- need more component tests  
+- need more action, reducer, selector tests  
+- many packages in package.json with caret (^) version, which can cause future incompatabilities between packages and therefore build failures, also potential security problems by automatically upgrading packages without code review. also, npm audit reveals 13 vulnerabilities  
+
+## Accessibility
+### Lighthouse scan
+#### score
+- 87  
+#### Names and Labels
+- buttons do not have an accessible name  
+#### Contrast
+- background and foreground colors do not have a sufficient contrast ratio   
+### My Manual Check Using Chrome
+- font size and icons may be too small in some places, especially in My Reading List and book grid  
```

Comment:
The accessibility check here would be whether `comand-+/-` work as expected rather than changing the font size.

Reply:
checked that increasing/decreasing page size using `ctrl-+/-` works